common/softwarecontainer-common.cpp

```diff
@@ -225,6 +259,44 @@ ReturnCode FileToolkitWithUndo::bindMount(const std::string &src, const std::str
     return ReturnCode::SUCCESS;
 }
 
+ReturnCode FileToolkitWithUndo::overlayMount(
+        const std::string &lower
```

Comment:
The style guidelines says the comma should be displaced so the following code aligns with the first param.